### DISCUSSION TITLE
Timer->Start playback error

### DISCUSSION THREAD

**Discussion by vernonjvs** [jcorporation/myMPD#624] [Q&A] [answered]
Using myMPD on a Raspberry Pi Zero W running Raspberry PI OS
```
uname -a 
Linux bose 5.10.63+ #1496 Wed Dec 1 15:57:05 GMT 2021 armv6l GNU/Linux

```
Built myMPD as per instructions
```
myMPD 8.1.6
(c) 2018-2021 Juergen Mang <mail@jcgames.de>
https://github.com/jcorporation/myMPD

```

Everything works great except for Timer->Start Playback.  I see the following errors in the log.

```
Dec 09 16:35:01 bose mympd[5451]: WARN     mympdapi  JSON path "$.params.plist" not found or value is not string type, found type is "MJSON_TOK_INVALID"
Dec 09 16:35:01 bose mympd[5451]: ERROR    mympdapi  Error processing method "INTERNAL_API_TIMER_STARTPLAY"
```
Any help would be greatly appreciated.

Timer->Stop playback works fine.

**Comment by jcorporation** [comment, marked as answer]
Thanks for reporting.
This seems to be a bug in the v8.1.6 release. Will be fixed for v8.2.0.

**Reply by vernonjvs** [reply]
Thanks very much for your response. I noticed these error messages on myMPD startup. I don''t know if they are a related to the issue.
```
03:15:24 ERROR    mpdworker Can not open file "/var/lib/mympd/smartpls/myMPDsmart-Genre-Country/Pop.rLC602" for write
03:15:24 ERROR    mpdworker Not a directory
03:15:24 ERROR    mpdworker Can not open file "/var/lib/mympd/smartpls/myMPDsmart-Genre-Dance & DJ/Electronica.Ch0aZU" for write
03:15:24 ERROR    mpdworker Not a directory
03:15:24 ERROR    mpdworker Can not open file "/var/lib/mympd/smartpls/myMPDsmart-Genre-Electronica/Dance.50NFgN" for write
03:15:24 ERROR    mpdworker Not a directory
03:15:24 ERROR    mpdworker Can not open file "/var/lib/mympd/smartpls/myMPDsmart-Genre-Folk/Rock.GeRBvJ" for write
03:15:24 ERROR    mpdworker Not a directory
03:15:24 ERROR    mpdworker Can not open file "/var/lib/mympd/smartpls/myMPDsmart-Genre-Pop Rock/Vocal.dUdg6F" for write
03:15:24 ERROR    mpdworker No such file or directory
03:15:24 ERROR    mpdworker Can not open file "/var/lib/mympd/smartpls/myMPDsmart-Genre-Trip-Hop / Downtempo.Cv6m1B" for write
03:15:24 ERROR    mpdworker No such file or directory
03:15:24 ERROR    mpdworker Can not open file "/var/lib/mympd/smartpls/myMPDsmart-Genre-Trip-Hop / Downtempo / Electronic.1OnLww" for write
03:15:24 ERROR    mpdworker No such file or directory
03:15:31 NOTICE   mpdworker Stopping mpd_worker thread
```
Also, when I Update smart playlists, I get the following error messages.
```
03:19:40 ERROR    webserver Unknown API method
03:19:40 ERROR    webserver Invalid API request
```
Hope this helps. Thanks

**Reply by jcorporation** [reply]
You found two another bugs:
- #628 
- #629

**Reply by vernonjvs** [reply]
Thanks. Confirmed #628 and #629 fixed in development branch. With regard to Timer->Start Playback, I get the following warning and different Error message in the 8.2 development branch when I click on the Playlist->Database drop down and I now don't see any playlists. Please note that this is a different error message from the one I preciously reported in 8.1.6. where the Playlist->Database drop down did show playlists.
 ```
17:23:27 WARN     mympdapi  JSON path "$.params.limit" not found
17:23:27 ERROR    mympdapi  Error processing method "MYMPD_API_PLAYLIST_LIST"

```
Thanks for all your incredibly fast responses. They are much appreciated.

**Reply by jcorporation** [reply]
Please star myMPD if you like it

**Reply by vernonjvs** [reply]
I don't know Github very well but I believe I have starred your excellent project.

**Comment by vernonjvs** [comment]
I believe that all of my reported bugs have been corrected in the development build. Thanks very much.
